app/Models/Subsidiary.php

```
@@ -64,23 +64,28 @@ public function deleteSubsidiary($id) {}
-                $query->where('sub_per_branch', $branch->branch_code);
-            })->when(isset($date), function ($query) use ($date) {
-                $query->whereDate('sub_date', '<=', $date);
-            })->whereHas('subsidiary_category', function ($query) {
-                $query->where('sub_cat_type', 'depre');
-            })->whereNotNull('sub_per_branch')->with(['subsidiary_accounts'])->get();
+        try {
+            $subsidiary = Subsidiary::when($categoryId, function ($query) use ($categoryId) {
+                $query->where('sub_cat_id', $categoryId);
+            })->with(['depreciation_payments' => function ($query) use ($date) {
+                $query->when($date, function ($query) use ($date) {
+                    $month = $date->month;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$month'. (Unused Code Rules)

(UnusedLocalVariable)